server/service/integration_mdm_test.go

```diff
@@ -5662,6 +5661,14 @@ func (s *integrationMDMTestSuite) TestBootstrapPackage() {
 	s.DoJSON("DELETE", "/api/latest/fleet/mdm/bootstrap/0", nil, http.StatusNotFound, &deleteResp)
 }
 
+func (s *integrationMDMTestSuite) TestBootstrapStatusFilterOnlyMacOS() {
+	t := s.T()
+	pkg, err := os.ReadFile(filepath.Join("testdata", "bootstrap-packages", "signed.pkg"))
+	require.NoError(t, err)
+
+	s.uploadBootstrapPackage(&fleet.MDMAppleBootstrapPackage{Bytes: pkg, Name: "pkg.pkg", TeamID: 0}, http.StatusOK, "")
```

Comment:
Is that test complete? Looks like you intended to test the filtering but I think this just uploads the bootstrap package.

Reply:
Lol yeah my bad! I started this test but then found a better place imo to do it in an existing test. Removing!